bot/general.py

```diff
@@ -13,12 +13,28 @@
 from bot.social.profile import Profile
 from bot.utils import paginate, PaginatedEmbedManager
 
+from bot.across_the_stars.vote import Vote
+
 
 class GeneralCog(commands.Cog):
     """
     Miscelânea
     """
 
+    emoji_answers_vote = {
+        '1': '1️⃣',
+        '2': '2️⃣',
+        '3': '3️⃣',
+        '4': '4️⃣',
+        '5': '5️⃣',
+        '6': '6️⃣',
+        '7': '7️⃣',
+        '8': '8️⃣',
+        '9': '9️⃣',
+        '10':'🔟'
+    }
```

Comment:
Pela natureza dessa lista de emojis ser totalmente numérica, podemos substituir esse dicionário por uma lista mesmo:
```suggestion
    emoji_answers_vote = [
        '1️⃣',
        '2️⃣',
        '3️⃣',
        '4️⃣',
        '5️⃣',
        '6️⃣',
        '7️⃣',
        '8️⃣',
        '9️⃣',
        '🔟'
    ]
```